### PR TITLE
Cancel transfer

### PR DESCRIPTION
**Pull request by ghiobi**
A user may cancel a transfer and have it re-deposit back into the sender's wallet.

### CODE REVIEW

File: tests/feature/TransferTest.php [hunk 1]
```diff
@@ -119,4 +119,34 @@ public function testReceiveTransfer()
...
+            'amount_display' => '$10.00'
+        ]);
+
+        $this->post('api/transfer/user/cancel', [
+            'token' => $transfer->token
+        ])->assertSee('data');
+
+        $wallet = $user->wallets()->where('currency_code', 'CAD')->first();
+
+        $this->assertEquals('11000', $wallet->balance);
```
Comment (Scieon) [inline tests/feature/TransferTest.php]: Just want to make sure, so the user has $11.00 before creating the transfer right?

Comment (ghiobi) [inline tests/feature/TransferTest.php]: The user had $100.00 before canceling the transfer. so ten dollars when back to their account.

Comment (Scieon) [inline tests/feature/TransferTest.php]: I was just testing you. Congratulations you have passed!

Comment (ghiobi) [inline tests/feature/TransferTest.php]: > The question of whether a computer can think is no more interesting than the question of whether a submarine can swim. - Edsger W. Dijkstra

**Pull request merged**